.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishSetExtractorConfiguration.cs

```diff
@@ -29,6 +29,9 @@ public class EnglishSetExtractorConfiguration : BaseDateTimeOptionsConfiguration
         public static readonly Regex EachDayRegex =
             new Regex(DateTimeDefinitions.EachDayRegex, RegexFlags);
 
+        public static readonly Regex BeforeEachDayRegex =
```

Comment:
Why give it a different name? I see in other languages it is its own regex. 
If the reuse of DayRegex is correct, just add a code comment here explaining.

Reply:
Only for the English set extractor do I want to use DayRegex instead of BeforeEachDayRegex.